pc_setup_user.sh

```
@@ -1,7 +1,7 @@
 cd
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)